CONTRIBUTING.md

```diff
@@ -137,7 +137,7 @@ However, there might be a way to implement that feature *on top of* Docker.
   <tr>
     <td>Stack Overflow</td>
     <td>
-      Stack Overflow has over 17000 Docker questions listed. We regularly
+      Stack Overflow has over 27 thousand Docker questions listed. We regularly
```

Comment:
I meant changing form such a specific number like 27k, to simply "Stack Overflow has thousands of Docker questions listed."